src/app/scripts/cac/control/cac-control-sidebar-directions.js

```diff
@@ -102,21 +114,20 @@ CAC.Control.SidebarDirections = (function ($, Control, BikeOptions, Geocoder, Ma
         setDirections: setDirections
     };
 
-    return SidebarDirectionsControl;
-
-    function changeMode() {
```

Comment:
Were these functions deleted by accident?  `changeMode` and `clearDirections` seem to still be needed.